src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -1911,6 +1923,38 @@ public Visitable processMethodDeclaration(MethodDeclaration node) {
     return result;
   }
 
+  /**
+   * Processes annotations by generating synthetic classes for unresolved annotations
+   *
+   * @param annotations The annotations to resolve
+   */
+  public void processAnnotations(NodeList<AnnotationExpr> annotations) {
+    for (AnnotationExpr anno : annotations) {
+      // If we can resolve the annotation, we do not need to generate a synthetic version
+      try {
+        anno.resolve();
+      } catch (UnsolvedSymbolException ex) {
+        UnsolvedClassOrInterface unsolvedAnnotation =
+            updateUnsolvedClassWithClassName(anno.getNameAsString(), false, false);
+
+        unsolvedAnnotation.setIsAnAnnotationToTrue();
+
+        // Add annotation parameters, but resolve the annotation parameters to Object;
+        // it doesn't matter what type they are synthetically generated as
+        if (anno.isNormalAnnotationExpr()) {
+          for (MemberValuePair pair : anno.toNormalAnnotationExpr().get().getPairs()) {
+            unsolvedAnnotation.addMethod(
+                new UnsolvedMethod(
+                    pair.getNameAsString(), "Object", Collections.emptyList(), true));
+          }
+        } else if (anno.isSingleMemberAnnotationExpr()) {
+          unsolvedAnnotation.addMethod(
+              new UnsolvedMethod("param1", "Object", Collections.emptyList(), true));
```

Comment:
We should use `value`, following [JLS 9.7.3](https://docs.oracle.com/javase/specs/jls/se8/html/jls-9.html#jls-9.7.3).